lib/rubocop/cop/style/multiple_comparison.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks against comparing a variable with multiple items,
+      # where `Array.include?` could be used instead to avoid DRY errors.
```

Comment:
If only we had `Set` literals. :) But given that you need an array anyway to create a `Set` it's probably not worth it - I expect people to have a few items at most, for which a simple array will do just fine.

But yeah, there is definitely some room for configuration. For example, if you use Rails/Activesupport you will have `Object#in?` which makes this much more readable.